src/styles/components/_landingLayout.scss

```diff
@@ -15,3 +15,34 @@
     width: 100%;
   }
 }
+
+.landing-sub-container {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+  min-width: 400px;
+  text-align: center;
+}
+
+.landing-logo { margin-top: 70px; }
+
+.landing-header {
+  letter-spacing: 3px;
+  line-height: 27px;
+  margin-bottom: 13px;
+  margin-top: 16px;
+}
+
+.landing-subtitle {
+  line-height: 32px;
+  margin: 0 0 10px 0;
```

Comment:
are those 0s overriding something? if not you can just do: `margin-bottom: 10px;`

Reply:
Yes, I am overriding the `margin-top` property. Maybe I can do:
```
{
    margin-bottom: 10px;
    margin-top: 0;
}
```

Reply:
`margin: 0 0 10px` then